samples/matter/template/README.rst

```diff
@@ -115,35 +115,28 @@ Device Firmware Upgrade support
 
 .. matter_template_nrf54l15_build_with_dfu_start
 
-The Device Firmware Upgrade (DFU) for the nRF54L15 PDK is exclusively available for the ``release`` build configuration and is limited to using the internal MRAM for storage.
-This means that both the currently running firmware and the new firmware to be updated must be stored within the device's internal flash memory.
-Currently, there is no support for utilizing external flash memory for this purpose.
+Similarly to other supported platforms, the Device Firmware Upgrade (DFU) for the nRF54L15 PDK is supported in a configuration that stores the upgraded image in the external FLASH memory.
```

Comment:
TBH I would remove this `matter_template_nrf54l15_build_with_dfu` section. It was added to explain that only release configuration supports it and it uses some new MRAM type of memory. Now this section does not differ from any other platform, it is just supported with external flash for both configurations, so maybe there is no point to describe it separately.

Reply:
Yes, that's probably a good idea. @wiba-nordic any comment on that?

Reply:
Yes, I agree, we can remove this section since it was a kind of workaround for the Devtag release.

Reply:
Remember to remove a link to this section from other samples.